plutus-ledger-api/test-plugin/Spec/Data/Budget/9.6/geq1.budget.golden

```diff
@@ -0,0 +1,2 @@
+({cpu: 739705185
+| mem: 1839010})
```

Comment:
Non-data version:
```
({cpu: 234326645
| mem: 870930})
```

Reply:
A 3.15x reduction in CPU and a 2.11x reduction in MEM! That's pretty cool.

Reply:
I wish that were true, unfortunately it's the other way around @effectfully @bezirg .

As I mentioned in the description, I wasn't able yet to find a way to optimise `geq` and the other comparison functions the same way I did with `eq` (which does show some improvement, as you noticed!).
I will most definitely come back to this, but I decided to not dwell on it longer than a couple of days because I ultimately don't know how much `geq` etc. are really used in practice.